### DISCUSSION TITLE
New runelite border, font and icon?

### DISCUSSION THREAD

**Discussion by AIENGI** [runelite/runelite#17435] [Q&A] [answered]
https://imgur.com/tsRkLMy one of my runelite clients crashed and when I reloaded the font, icon and border are completely different. Is this an unlisted update or what is going on here? I lost 6 hours of shaman kc on the loot tracker which isn't important but now the client looks way different. Is this something I can change or was this just an update I missed

**Comment by raiyni** [comment, marked as answer]
Runelite uses a new look and feel once you restart your client.

**Reply by AIENGI** [reply]
Thanks. Just wanted to make sure it wasn't some sort of issue as I hadn't seen any notes on the new look. Appreciate you putting my anxiety to rest! lol
